### PR TITLE
Fix validation rules

### PR DESCRIPTION
**Pull request by Zaknafeyn**

### CODE REVIEW

File: public/scripts/submit.js [hunk 1]
```diff
@@ -1,11 +1,31 @@
...
+      submitButt = $("button[type='submit']");
+
+  checkboxes.click(function(e) {
+    submitButt.prop("disabled", !checkboxes.is(":checked"));
+  });
+
+  submitButt.Click(function (e){
+    if (!checkboxes.is(":checked")) {
+      // show error message
+      // toastr.error("Select course you want to participate.");
```
Comment (sudodoki) [inline public/scripts/submit.js]: Not sure we need dead code here

**Comment by sudodoki** [comment]
Great work! :heart_eyes_cat: 
Would suggest removing lines 12-16 with `submitButt.Click` stuff.

**Comment by Zaknafeyn** [comment]
Unused code was removed.
Add 3 new coaches to main board

**Comment by sudodoki** [comment]
>Add 3 new coaches to main board
Please, do as a separate commit

**Pull request closed without merging**